tests/pages/services/ServiceActions-cy.js

```diff
@@ -219,13 +219,14 @@ describe('Service Actions', function () {
         .route({
           method: 'PUT',
           url: /marathon\/v2\/apps\/\/cassandra\-healthy/,
-          response: [],
-          delay: 50
-        });
+          response: []
+        }).as('marathon/apps/cassandra-healthy:scaleService');
+
       cy.get('.modal-footer .button-collection .button-primary')
         .as('primaryButton').click();
-      cy.get('@primaryButton').should('have.class', 'disabled');
```

Comment:
Same here, we're actually testing that the button is disabled when waiting for the request response.